backend/gce/metadata.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
Can you remove this extra build statement

Reply:
removed the above one to remain consistent with style